src/Components/MessageState/EmptyStates.js

```diff
@@ -87,4 +90,34 @@ const NoMatchingRecs = () => {
   );
 };
 
-export { ErrorState, NoAffectedClusters, NoMatchingClusters, NoMatchingRecs };
+// used in Routes.js to create custom message instead of <InvalidObject>
+const ComingSoon = () => {
+  const intl = useIntl();
+  return (
+    <EmptyState variant="small">
+      <Spinner />
+      <Title headingLevel="h2" size="2xl">
+        {intl.formatMessage(messages.comingSoonTitle)}
+      </Title>
+      <EmptyStateBody>
+        {intl.formatMessage(messages.comingSoonBody)}
+      </EmptyStateBody>
+      <Button
+        component="a"
```

Comment:
Use the `Link` component from the react-router-dom and pass `Button` as a `component` property (https://v5.reactrouter.com/web/api/Link/component-reactcomponent)